bond/package.xml

```diff
@@ -1,31 +1,34 @@
 <?xml version="1.0"?>
-<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>bond</name>
-  <version>1.8.3</version>
+  <version>1.8.4</version>
```

Comment:
This is probably worth a major version bump.

Reply:
To 2.0.0

Reply:
@mjcarroll 
I have changed the version to 2.0.0